server/notification-providers/slack.js

```diff
@@ -49,7 +49,75 @@ class Slack extends NotificationProvider {
                 return okMsg;
             }
 
+            const actions = [];
```

Comment:
Please extract the action generation into another function to not make this function too large